### ISSUE TITLE
[documentation only] minor error in sample code

### ISSUE THREAD

**Issue by warren-bank** [remix-run/remix#8653]
### Reproduction

https://github.com/remix-run/remix/blob/main/docs/file-conventions/remix-config.md#servernodebuiltinspolyfill

### System Info

```shell
N/A
```

### Used Package Manager

npm

### Expected Behavior

```javascript
typeof module.exports.serverNodeBuiltinsPolyfill.globals === 'object'
typeof module.exports.globals === 'undefined'
```

### Actual Behavior

```javascript
typeof module.exports.serverNodeBuiltinsPolyfill.globals === 'undefined'
typeof module.exports.globals === 'object'
```

**Comment by brophdawg11** [comment]
Can you elaborate a bit?  I'm not sure quite what you're suggesting the docs be updated to?

**Comment by warren-bank** [comment]
sorry.. I thought that would be sufficiently clear;
please correct me if I'm wrong, as I'm new to this framework..
but it would seem to me [that](https://github.com/remix-run/remix/blob/main/docs/file-conventions/remix-config.md?plain=1#L213-L226):

```js filename=remix.config.js
/** @type {import('@remix-run/dev').AppConfig} */
module.exports = {
  serverNodeBuiltinsPolyfill: {
    modules: {
      buffer: true, // Provide a JSPM polyfill
      fs: "empty", // Provide an empty polyfill
    },
  },
  globals: {
    Buffer: true,
  },
};
```

should be:

```js filename=remix.config.js
/** @type {import('@remix-run/dev').AppConfig} */
module.exports = {
  serverNodeBuiltinsPolyfill: {
    modules: {
      buffer: true, // Provide a JSPM polyfill
      fs: "empty", // Provide an empty polyfill
    },
    globals: {
      Buffer: true,
    },
  },
};
```

**Comment by brophdawg11** [comment]
ahh ok even with that side by side it took me a while to spot the difference!  Just fixed in a5d10eb1efa0358eeadfb9e5aaa1190ebf8d7707.  Thanks for the callout!
